src/views/join/join.jsx

```diff
@@ -1,14 +1,15 @@
 const React = require('react');
 const render = require('../../lib/render.jsx');
-const JoinModal = require('../../components/modal/join/modal.jsx');
+const Scratch3Registration = require('../../components/registration/scratch3-registration.jsx');
 const ErrorBoundary = require('../../components/errorboundary/errorboundary.jsx');
 // Require this even though we don't use it because, without it, webpack runs out of memory...
 const Page = require('../../components/page/www/page.jsx'); // eslint-disable-line no-unused-vars
 
 const openModal = true;
 const Register = () => (
     <ErrorBoundary>
-        <JoinModal
+        <Scratch3Registration
```

Comment:
I'm curious why you switched to using Scratch3Registration here rather than just passing the new prop directly to the JoinModal.  I had a avoiding using Scratch3Registration before just because it had more functionality than I thought we needed, but I'm also still learning how to do things in React.

Reply:
This is a good question! Scratch3Registration is wired up to redux, so we can use the navigation.js library to handle what to do when the user clicks the final button at the very end, rather than adding a copy of this functionality to join.jsx or JoinModal.

Reply:
Makes sense!  This seems good!

At some point, I'd love to discuss whether it might make sense to send a function or maybe the button itself? as a prop instead of a bool. Then the function or button would just do the right thing when it was clicked and the welcome step wouldn't be concerned with anything.  It feels like that might be a more generic way to do things that keeps the components at the bottom of the tree simpler.  Just thinking out loud and wondering what best practices are w.r.t. React design though.